fgread/read.py

```diff
@@ -292,21 +296,35 @@ def load_data(
 
     exp_count = single_df.loc[0, "numberOfExpressionDataFiles"]
     meta_count = single_df.loc[0, "numberOfMetaDataFiles"]
+
     if exp_count == 0:
         raise TypeError(
             f"There is no expression data available in this data set.\n"
             f"Metadata files: {meta_count}."
         )
-    elif exp_count >= 2:
-        raise TypeError(
-            f"There are {exp_count} expression data files and {meta_count} metadata files in this dataset. "
-            "Currently we only provide reading functionality for one expression data file and ignore meta data. "
-            "Please load the required data yourself."
-        )
+
+    exp_files = [exp["name"] for exp in single_df.loc[0, "expressionDataFileInfos"]]
+
+    if expression_file:
+        if expression_file in exp_files:
+            file = expression_file
+        else:
+            raise KeyError(
+                f"Expression file {expression_file} not found in dataset. "
+                f"Available expression files are: {exp_files}."
+            )
+    else:
+        if exp_count == 1:
+            file = single_df.loc[0, "expressionDataFileInfos"][0]["name"]
+        else:
+            raise TypeError(
+                f"There are {exp_count} expression data files in this dataset. "
+                'Please specifiy which you want to load by setting "expression_file". '
```

Comment:
Vorschlag: Statt
`Please specifiy which you want to load by setting "expression_file`
sollten wir schreiben
`Please specify which one you want to load using the parameter "expression_file`
(vorne ein Typo, hinten Parameter -> ist m.E. für Python User klarer als das Wort "setting")